benchmarks/query_middleware.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+var http = require('http')
+var Benchmark = require('benchmark')
+var suite = new Benchmark.Suite()
+
+var sr = {}
+
+var middleware = require('../lib/middleware/query')()
```

Comment:
I'm not 100% sure what the purpose for the benchmark of the `query` middleware is for; it seems like it's pretty much just benching the `qs` module performance, no? The `query` middleware no longer exists in Express 5.x anyway :)